### ISSUE TITLE
Need some help in running the repo.

### ISSUE THREAD

**Issue by SecByShresth** [tensorflow/nmt#317]
I need some help in running the program, I followed all the steps and trained this to 1000 steps. Now when I am running this so that I can get the required output, and can check the results. I am just getting the output as FAILED (failures = 3). So how to resolve this. P.S. I'm using the following command. 

python -m nmt.model_test

![image](https://user-images.githubusercontent.com/14314216/39300631-f9ea7bbc-4969-11e8-865e-03fdcc936da1.png)

**Comment by shresthpaul133** [comment]
@oahziur Sir please help.

**Comment by sanjibnarzary** [comment]
To train the vi to en training set follow the following steps

1. Install tensorflow nightly
`pip install -U tf-nightly`
2. Clone nmt repository
`git clone https://github.com/tensorflow/nmt/`
3. `cd nmt`
4. ` nmt/scripts/download_iwslt15.sh /tmp/nmt_data`
5. `mkdir /tmp/nmt_model`
6. `python -m nmt.nmt \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu`
7. Done

**Comment by shresthpaul133** [comment]
@oahziur Sir any idea on how to tackle this problem ? I corrected the data and trained it on my own data set.

**Comment by ranjita-naik** [comment]
I also ran into similar issues and I resolved it making these changes https://github.com/tensorflow/nmt/pull/401
